RNWCPP/ReactWindows.nuspec

```diff
@@ -29,6 +29,16 @@
     <file src="inc\cxxreact\NativeToJsBridge.h"      target="inc\cxxreact" />
     <file src="inc\cxxreact\NativeModule.h"          target="inc\cxxreact" />
 
+    <file src="inc\jsi\jsi.h"                        target="inc\jsi" />
+    <file src="inc\jsi\jsi-inl.h"                    target="inc\jsi" />
+    <file src="inc\jsi\ScripStore.h"                 target="inc\jsi" />
+    <file src="inc\jsi\RuntimeHolder.h"              target="inc\jsi" />
+
+    <file src="inc\jschelpers\JavaScriptCore.h"      target="inc\jschelpers" />
+    <file src="inc\jschelpers\JSCWrapper.h"          target="inc\jschelpers" />
+    <file src="inc\jschelpers\Unicode.h"             target="inc\jschelpers" />
```

Comment:
I thought @acoates-ms removed unicode.h recently on purpose that it was going away